packages/react-native-sdk/src/components/participants/internal/ParticipantActions.tsx

```diff
@@ -176,11 +177,14 @@ export const ParticipantActions = (props: ParticipantActionsType) => {
         ]
       : [];
 
-  const pinParticipant: CallParticipantOptionType | null = {
-    icon: <Pin color={theme.dark.text_high_emphasis} />,
-    title: participant.pinnedAt ? 'Unpin' : 'Pin',
-    onPressHandler: toggleParticipantPinnedAt,
-  };
+  const pinParticipant: CallParticipantOptionType | null =
+    !participant.pin || participant.pin.isLocalPin
```

Comment:
Is there a way we can simplify this check? 🤔

Reply:
fixed.